cfc-testing/master-validation-system.sh

```
@@ -192,7 +192,7 @@ run_smoke_tests() {
         local ssl="${CONFIG_PARTS[4]}"
         local subdomain="${CONFIG_PARTS[5]}"
         
-        local stack_name="smoke-$(echo "$config" | tr '[:upper:],' '[:lower:]-' | tr -d ',')"
+        local stack_name="smoke-$(echo "$config" | tr '[:upper:],' '[:lower:]-' | tr -d ',' | tr '_' '-')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)